docs/en_US/NAS/retiarii/Tutorial.rst

```diff
@@ -0,0 +1,171 @@
+Neural Architecture Search with Retiarii (Experimental)
+=======================================================
+
+`Retiarii <https://www.usenix.org/system/files/osdi20-zhang_quanlu.pdf>`__ is a new framework to support neural architecture search and hyper-parameter tuning. It allows users to express various search space with high flexibility, to reuse many SOTA search algorithms, and to leverage system level optimizations to speed up the search process.
+
```

Comment:
The support of NAS and hyper-parameter tuning looks similar to NNI? Why highlight the new framework?

Reply:
this is a good question. it provide new user experience, experiment is launched in python code, search space can be expressed directly in user model code, supports both neural architecture space and hyper-parameter space. will make it clear in this doc